qualtran/bloqs/mcmt/multi_control_multi_target_pauli.py

```diff
@@ -123,20 +123,25 @@ def decompose_from_registers(
         self, *, context: cirq.DecompositionContext, **quregs: NDArray['cirq.Qid']
     ) -> cirq.OP_TREE:
         controls, target = quregs.get('controls', np.array([])), quregs['target']
-        if len(self.cvs) <= 2:
+        if len(self.cvs) < 2:
```

Comment:
do you still need `controls.flatten()`? Does this work for len(self.cvs) == 0?

Reply:
Yes and Yes. We have tests already for `len(self.cvs) == 0` which are passing after this PR as well.